.github/workflows/Release.yml

```diff
@@ -37,11 +37,14 @@ jobs:
           validate-wrappers: true
           gradle-home-cache-cleanup: true
 
-      - name: Install GMD image for baseline profile generation
-        run: yes | "$ANDROID_HOME"/cmdline-tools/latest/bin/sdkmanager "system-images;android-33;aosp_atd;x86_64"
+      - name: Setup Android SDK
+        uses: android-actions/setup-android@v3
 
-      - name: Accept Android licenses
-        run: yes | "$ANDROID_HOME"/cmdline-tools/latest/bin/sdkmanager --licenses || true
+      - name: Accept licenses
+        run: yes | sdkmanager --licenses || true
+
+      - name: Setup AVD for GMD use
+        run: yes | sdkmanager "system-images;android-33;aosp_atd;x86_64" || true
```

Comment:
GMD will download system images for you, so you don't have to install it manually here.

Reply:
Good point. Removed